install.sh

```
@@ -1,102 +1,370 @@
   echo "Downloading and installing CeraUI content"
-  # Create a temporary directory for CeraUI
-  CERAUI_TEMP_DIR="$HOME/.tmp/ceraui"
-  mkdir -p "$CERAUI_TEMP_DIR"
-  cd "$CERAUI_TEMP_DIR" || exit
 
-  # Download and extract CeraUI
-  wget -q --show-progress $CERAUI_RELEASE_URL
-  tar xf $CERAUI_RELEASE_TARBALL
+  if [[ "$DEPLOY_MODE" == "remote" ]]; then
+    # Create a temporary script to download and extract CeraUI on the remote machine
+    local tmp_script=$(cat <<'EOF'
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)